applications/ContactStructuralMechanicsApplication/python_scripts/contact_structural_mechanics_implicit_dynamic_solver.py

```diff
@@ -1,11 +1,26 @@
 from __future__ import print_function, absolute_import, division  # makes KM backward compatible with python 2.6 and 2.7
 #import kratos core and applications
 import KratosMultiphysics as KM
+
```

Comment:
I guess we shall best wait for this to be merged to the incoming changes to the solvers...